sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java

```diff
@@ -28,11 +29,33 @@ static void configureExporter(
       case "prometheus":
         configurePrometheusMetrics(config, meterProvider);
         return;
+      case "logging":
+        ClasspathUtil.checkClassExists(
+            "io.opentelemetry.exporter.logging.LoggingMetricExporter",
+            "Logging Metrics Exporter",
+            "opentelemetry-exporter-logging");
+        configureLoggingMetrics(config, meterProvider);
+        return;
       default:
         return;
     }
   }
 
+  private static void configureLoggingMetrics(
+      ConfigProperties config, SdkMeterProvider meterProvider) {
+    LoggingMetricExporter exporter = new LoggingMetricExporter();
+    IntervalMetricReaderBuilder readerBuilder =
+        IntervalMetricReader.builder()
+            .setMetricProducers(Collections.singleton(meterProvider))
+            .setMetricExporter(exporter);
+    Long exportIntervalMillis = config.getLong("otel.imr.export.interval");
```

Comment:
Can we extract a method for creating an IMR given an exporter?

Reply:
done